docs/src/technik/get_started/index.md

```diff
@@ -4,4 +4,16 @@
 
 Im Projekte verwenden wir `checkstyle` und `spotless` um für einen möglichst einheitlichen Codestyle zu sorgen.
```

Comment:
"Im Projekt" oder "In Projekten" anstatt "Im Projekte"

Reply:
Habe ich zum Anlass genommen noch einmal durch die gesamte Doku auf den `dev`-Branch zu gehen und nach Typos zu suchen. Daraus entstand ein neuer PR: #78